src/templates/finops-hub/modules/storage.bicep

```
@@ -36,6 +36,9 @@ param msexportRetentionInDays int = 0
 @description('Optional. Number of months of cost data to retain in the ingestion container. Default: 13.')
 param ingestionRetentionInMonths int = 13
 
+@description('Optional. Enable infrastructure encryption on the storage account.')
+param enableInfrastructureEncryption bool = true
```

Update the default and description here if you end up changing the main.bicep version.

Small nit: Can you move this above the tags parameter? I'd like to keep the storage account configuration settings together.

```suggestion
 @description('Optional. Enable infrastructure encryption on the storage account. Default = false.')
param enableInfrastructureEncryption bool = false
```